homeassistant/components/light/plum_lightpad.py

```diff
@@ -0,0 +1,186 @@
+"""
+Support for Plum Lightpad switches.
+
+For more details about this component, please refer to the documentation at
+https://home-assistant.io/components/light.plum_lightpad
+"""
+from homeassistant.components.light import (
+    ATTR_BRIGHTNESS, ATTR_HS_COLOR, SUPPORT_BRIGHTNESS, SUPPORT_COLOR, Light)
+from homeassistant.components.plum_lightpad import PLUM_DATA
+import homeassistant.util.color as color_util
+
+DEPENDENCIES = ['plum_lightpad']
+
+
+async def async_setup_platform(hass, config, async_add_entities,
+                               discovery_info=None):
+    """Setup the Plum Lightpad Light."""
+    plum = hass.data[PLUM_DATA]
+
+    if discovery_info is not None:
```

Comment:
Make this a guard clause instead by inverting it and return if true. And put it first in the function body.